examples/lds01/include/lds01_params.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
how the choice is made between this file and the default driver one ?

Reply:
Include directories from applications and examples are included before drivers includes (cf RIOT driver guidelines).